src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -107,6 +108,12 @@ private OrbitInfo GetNextPatch()
             return ! GetHasNextPatch() ? null : new OrbitInfo(orbit.nextPatch,Shared);
         }
 
+
+        private double GetNextPatchETA()
+        {
+            return  ! GetHasNextPatch() ? Double.PositiveInfinity : orbit.EndUT - Planetarium.GetUniversalTime();
```

Comment:
Since any value of infinity is an invalid in kerboscript and will throw an exception, it would be better for this to default to either zero or -1.  I think I would prefer it to be zero.  And while it appears the syntax was copied from above, I think it's cleaner if remove the `!` conditional and reverse the `? ... : ...` parameters.